docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,197 @@
+    environment
+    FROM pmm."metrics" 
+    WHERE $__timeFilter(period_start) AND environment != ''
+    GROUP BY t, environment
+    ORDER BY t    
+    ```
+**Grafana**
+
+Grafana has been upgraded to 9.2.18 due to [CVE-2023-28119](https://github.com/advisories/GHSA-5mqj-xc49-246p) and [CVE-2023-1387](https://github.com/advisories/GHSA-c3h9-vpfv-3x4m).
+
+**Exporters** 
+
+Updated the mysqld_exporter, node_exporter, postgres_exporter, mongo_exporter to fix the CVE's.
+
+
+### Improved nodes view on the Inventory page
+
+In our continuing endeavor to enhance user experience, we further improved the **Inventory** page by improving the **Nodes** tab. With this change, you can get more context on your inventory objects, such as node, and more actionable information for exploring and resolving possible issues.
+
+Here is the change as part of PMM 2.38.0:
+
+- You can now see the number of agents running on any particular node. When you click on any node, the UI navigates to the view of agents, which is filtered to display only agents related to that specific node. 
+
+To see the details of the agents running, do the following:
+
+On the **Nodes** tab, under the **Monitoring** column, click **OK** or **Failed** depending on the status of the node that you have selected. A page that provides the user with crucial information regarding the total number of agents deployed on that node is displayed.
+    ![!image](../_images/PMM_Inventory_Node_Agent_Properties.png)
+
+- You can also view comprehensive information about each agent, including key attributes such as node type, IP address, and associated services. This gives you a complete overview of the nodes at a glance.
+
+    ![!image](../_images/PMM_Inventory_Node_Selection.png)
+
+
+- You can check the health status of the agents to see if they are running or have failed.
+
+### Backup enhancements
+
+#### Better folder management for enhanced PMB compatibility 
+We’ve added a new **Folder** field to the Create Backup pages. Use this to set a specific target directory within the selected local or S3-compatible location for the backup.  
+
+Organizing backups in folders makes grouping PBM backups for clusters easier, and it improves PMM-PBM (Percona Backup for MongoDB) integration workflows. 
+
+The **Folder** field is automatically populated with the value of the cluster label. You can change this default folder from PMM’s Advanced Sett`ings, but make sure you understand how your custom folder will impact PBM integration workflows.
+
+
+#### Sharded cluster MongoDB backup support 
+This release introduces support for working with backups for sharded clusters. PMM handles the backup process end-to-end, but restoring such artifacts is currently possible only via the CLI, using Percona Backup for MongoDB.
+
+For information on restoring sharded backups, check the [PBM documentation](https://docs.percona.com/percona-backup-mongodb/usage/restore.html)
+
+
+### SQL comments support in QAN
+
```

Comment:
Note: This is a technical preview feature. Link to definition https://docs.percona.com/percona-monitoring-and-management/details/glossary.html#technical-preview

Reply:
Will add it